src/main/java/com/facebook/bitEncoderDecoder/bitwise/BitwiseEncoderImpl.java

```diff
@@ -20,4 +20,16 @@ private String charHexToBin(Character ch) {
         }
         return result.append("0").append(bin).toString();
     }
+
+    private String getBinaryString(String input) {
+        try {
+            return Integer.toBinaryString(Integer.parseInt(input));
```

Comment:
you can use Integer.toBinaryString directly on chars, since they are numbers. Would you consider something like: for each char in String {charHexToBin}?